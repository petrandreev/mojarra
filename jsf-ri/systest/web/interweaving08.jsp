<%@page pageEncoding="UTF-8"%>
<%@taglib prefix="f" uri="http://java.sun.com/jsf/core"%>
<%@taglib prefix="h" uri="http://java.sun.com/jsf/html"%>


<f:view>
    <html>
        <body>
            <h:form id="form">
                <h:dataTable value="#{interweaving08.items}" var="item"
binding="#{interweaving08.table}">
                    <h:column>
                        <%-- 
                           Validates CASE 1 in UIComponentClassicTagBase when
                           using a component binding with a lifetime longer
                           than request
                        --%>
                        ciao
                        <h:outputText value="#{item}"/>
                    </h:column>
                </h:dataTable>                
            </h:form>
        </body>
    </html>
</f:view>

