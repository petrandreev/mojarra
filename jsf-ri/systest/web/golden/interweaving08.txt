





    <html>
        <body>
            
<form id="form" name="form" method="post" action="/jsf-systest/faces/interweaving08.jsp;jsessionid=b6a55da92c55d06c043b90f68e43" enctype="application/x-www-form-urlencoded">
<input type="hidden" name="form" value="form" />
<input type="hidden" name="javax.faces.ViewState" id="javax.faces.ViewState" value="j_id21:j_id22" />
<table>
<tbody>
<tr>
<td>
                        
                        ciao
                        Value</td>
</tr>
</tbody>
</table>

</form>
        </body>
    </html>


